CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
   ## v8.10.0
 
 
+  * **gRPC instrumentation**
+
+    The agent will now instrument [gRPC](https://grpc.io/) activity performed by clients and servers that use the [grpc](https://rubygems.org/gems/grpc) RubyGem. Instrumentation is automatic and enabled by default, so gRPC users should not need to modify any existing application code or agent configuration to benefit from the instrumentation. The instrumentation makes use of New Relic Distributed Tracing for a comprehensive overview of all gRPC traffic taking place across multiple monitored applications. The Ruby agent's use of Distributed Tracing is compatible with other programming language based New Relic agents that support gRPC.
```

Comment:
```suggestion
    The agent will now instrument [gRPC](https://grpc.io/) activity performed by clients and servers that use the [grpc](https://rubygems.org/gems/grpc) RubyGem. Instrumentation is automatic and enabled by default, so gRPC users should not need to modify any existing application code or agent configuration to benefit from the instrumentation. The instrumentation makes use of New Relic Distributed Tracing for a comprehensive overview of all gRPC traffic taking place across multiple monitored applications. The Ruby agent's use of Distributed Tracing is compatible with other New Relic language agents that support gRPC.
```

Reply:
I don't think one exists yet, but when a docs page is added for gRPC instrumentation, could you add a link to it in the Changelog?

I think the doc could have more details about external segments being generated by the client, transactions generated by the server and the attributes each publish.